aexpect/remote.py

```diff
@@ -1,14 +1,35 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# This code was imported from the avocado-vt project,
+#
+# virttest/remote.py
+# Original author: Michael Goldish <mgoldish@redhat.com>
+#
+# Copyright: 2016 IBM
+# Authors : Michael Goldish <mgoldish@redhat.com>
+
 """
 Functions and classes used for logging into guests and transferring files.
 """
+
+# disable too-many-* as we need them pylint: disable=R0912,R0913,R0914,R0915,C0302
```

Comment:
sure, no problem

Reply:
It imitates other modules for the time being ^^